Sources/ContainerCommands/Container/ProcessUtils.swift

```diff
@@ -20,6 +20,14 @@ import ContainerizationError
 import ContainerizationOS
 import Foundation
 
+struct StandardError: TextOutputStream, Sendable {
```

Comment:
With #680 you should be able to just use the logger. I think you mentioned a while ago that the `StderrLogHandler` type I introduced for that isn't needed as there's an existing swift-logging type for logging to stderr, this might be a good time to make that switch.

See also #642.